src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java

```diff
@@ -451,6 +449,11 @@ private EditorFormFieldSet generateEditorFormFieldSet(Set<String> processedPrope
             !fieldSetEditable,
             editorFormFields
         );
+
+        // Set correct target for fieldset itself which will be used in case the fieldset doesn't have fields i. e. mixin definition without properties
+        fieldset.setTarget(new EditorFormFieldTarget(nodeType.getItemsType(), nodeType.getName(), -0.1));
```

Comment:
Curious what does the -0.1 do?
But being said does that mean that target section never worked? (As it was not assigned?)

Reply:
it sets rank. It worked as long as you had at least one property defined on your definition.